ethers-contract/tests/abigen.rs

```diff
@@ -374,6 +374,23 @@ fn can_handle_unique_underscore_functions() {
     let _contract_call = ConsoleLogCalls::Log2(call);
 }
 
+#[test]
+fn can_handle_underscore_numeric() {
+    abigen!(
+        Test,
+        r#"[
+            _100pct(string)
+        ]"#
+    );
+    let call = _100PctCall("message".to_string());
+
+    let provider = Arc::new(Provider::new(MockProvider::new()));
+    let contract = Test::new(Address::default(), Arc::clone(&provider));
+    // NOTE: this seems to be weird behaviour of `Inflector::to_snake_case` which turns "100pct" ->
+    // "10_0pct"
+    let _call = contract._10_0pct("hello".to_string());
```

Comment:
Do we know why it becomes `10_0pct` and not `100_pct`? @mattsse

Reply:
Check the comment right above this code line! It's an issue in the Inflector crate, which we'll track and solve separately